tests/dind/priv.bats

```diff
@@ -42,9 +42,18 @@ function teardown() {
 
   verify_inner_cont_procfs_mnt $syscont_name $inner_cont_name /proc priv
 
-  docker exec "$syscont_name" sh -c "docker exec $inner_cont_name sh -c \"mount | grep sysfs\""
-  [ "$status" -eq 0 ]
-  [[ "$output" == "sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)" ]]
+  # Verify sysfs mount attributes are the expected ones. Notice that CentOS/Redhat
+  # enable SELinux by default so expected mount attributes may vary (i.e. "seclabel"
+  # super-block attribute may be present).
+  if ! mount | egrep -q "sysfs.*seclabel"; then
```

Comment:
Rather than `if-else`, I suggest using regex:

```
[[ "$output" == "sysfs on /sys type sysfs".+"rw".+"nosuid".+"nodev".+"noexec".+"relatime" ]]
```

I think it's cleaner.

Reply:
I had something like that before, but i changed it to make the verification process more explicit: i want to see the "seclabel" present if SELinix is enabled.

I'm making a small adjustment to this logic in my next patch though.